SpatialGDK/Source/SpatialGDK/Public/Utils/InspectionColors.h

```diff
@@ -3,115 +3,12 @@
 #pragma once
```

Comment:
There is no functional change in this code, I just moved it to a cpp file so it could be used in multiple places.